src/observable.js

```diff
@@ -114,6 +114,52 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  subscribe(observer) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: arguments[1],
```

Comment:
Not sure we need to support `subscribe(onValue, onError)` format. Supporting `subscribe({onValue, onError})` + `subscribe(onValue)` is enough IMO. Is this format from ES spec?